internal/oci/runtime_vm.go

```diff
@@ -733,6 +733,66 @@ func (r *runtimeVM) ContainerStats(ctx context.Context, c *Container, _ string)
 	return metricsToCtrStats(c, metrics), nil
 }
 
+func metricsToCtrStats(c *Container, m *cgroups.Metrics) *ContainerStats {
+	var (
+		cpu             float64
+		cpuNano         uint64
+		memUsage        uint64
+		memLimit        uint64
+		memPerc         float64
+		workingSetBytes uint64
+		netInput        uint64
+		netOutput       uint64
+		blockInput      uint64
+		blockOutput     uint64
+		pids            uint64
```

Comment:
Personal nit, and since you added workingSetBytes I'll ask, can you alphabetize these?  It's getting long enough where that would be useful during reviews at least.